### PR TITLE
Dual-Monitor-Tools: Add version 2.7

### PR DESCRIPTION
**Pull request by TheRandomLabs**
Dual Monitor Tools is a utility for users with dual or multiple monitor setups.

### CODE REVIEW

File: bucket/dual-monitor-tools.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            "Dual Monitor Tools"
+        ]
+    ],
+    "persist": [
+        "DmtMagicWords.xml",
+        "DmtSettings.xml"
+    ],
+    "checkver": "Dual Monitor Tools is ([\\d.]+)",
+    "autoupdate": {
+        "url": "https://sourceforge.net/projects/dualmonitortool/files/dualmonitortool/$version/DualMonitorTools-$version.zip"
```
Comment (Ash258) [inline bucket/dual-monitor-tools.json]:
```suggestion
        "url": "https://downloads.sourceforge.net/project/dualmonitortool/dualmonitortool/$version/DualMonitorTools-$version.zip"
```

File: bucket/dual-monitor-tools.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "2.7",
+    "description": "Utility for multiple monitor setups.",
+    "homepage": "http://dualmonitortool.sourceforge.net/",
+    "license": "GPL-3.0-only",
+    "url": "https://sourceforge.net/projects/dualmonitortool/files/dualmonitortool/2.7/DualMonitorTools-2.7.zip",
```
Comment (Ash258) [inline bucket/dual-monitor-tools.json]:
```suggestion
    "url": "https://downloads.sourceforge.net/project/dualmonitortool/dualmonitortool/2.7/DualMonitorTools-2.7.zip",
```

File: bucket/dual-monitor-tools.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "hash": "sha1:3f36099857c08349944dfa3016a65385d8844fc6",
+    "pre_install": [
+        "New-Item -Path \"$dir\\DmtMagicWords.xml\" -ItemType File | Out-Null",
```
Comment (Ash258) [inline bucket/dual-monitor-tools.json]:
```suggestion
        "if (-not (Test-Path \"$persist_dir\\DmtMagicWords.xml\")) { New-Item \"$dir\\DmtMagicWords.xml\" -ItemType File | Out-Null }",
```

File: bucket/dual-monitor-tools.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        "New-Item -Path \"$dir\\DmtSettings.xml\" -ItemType File | Out-Null"
```
Comment (Ash258) [inline bucket/dual-monitor-tools.json]:
```suggestion
        "if (-not (Test-Path \"$persist_dir\\DmtSettings.xml\")) { New-Item \"$dir\\DmtSettings.xml\" -ItemType File | Out-Null }",
```

File: bucket/dual-monitor-tools.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "2.7",
+    "description": "Utility for multiple monitor setups.",
```
Comment (Ash258) [inline bucket/dual-monitor-tools.json]:
```suggestion
    "description": "Set of utilities to help with multi monitor setup management.",
```

File: bucket/dual-monitor-tools.json (hunk 3 above)
Comment (TheRandomLabs) [reply bucket/dual-monitor-tools.json]: Oh, I forgot this is run on every update.

**Comment by TheRandomLabs** [comment]
All suggestions applied.

**Pull request merged**